todo/src/index.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import * as serviceWorker from "./serviceWorker";
+import { Provider } from "react-redux";
+import { createStore } from "redux";
+import { reducer } from "./reducers";
+import { BrowserRouter as Router } from "react-router-dom";
+
+const store = createStore(reducer);
+
+ReactDOM.render(
+  <Router>
```

Comment:
You typically should wrap the Router inside the Provider
Check this out from Redux
https://github.com/reduxjs/redux/blob/master/docs/advanced/UsageWithReactRouter.md